vmdb/app/models/miq_server/rhn_mirror.rb

```diff
@@ -1,4 +1,5 @@
 require 'linux_admin'
+require 'miq_apache'
```

Comment:
Yes, it's used a few few places in regards to the MiqServer managing apache.  In these cases, Rails autoloading would make calls to MiqApache autoload from vmdb/lib.  Since lib/util is not in the autoload paths, we need to require 'miq_apache' which will require the two files.

In the case of the black console, where @mkanoor needs the apache root path and service name values, we'll require miq_apache and have access to this information.